external/docker-integration-tests/src/test/scala/org/apache/spark/sql/jdbc/v2/V2JDBCTest.scala

```diff
@@ -193,6 +193,8 @@ private[v2] trait V2JDBCTest extends SharedSparkSession with DockerIntegrationFu
 
   test("SPARK-36895: Test INDEX Using SQL") {
     if (supportsIndex) {
+      val indexOptions = if (catalogName.equals("mysql")) "KEY_BLOCK_SIZE=10"
+        else if (catalogName.equals("postgresql")) "FILLFACTOR=70"
```

Comment:
Depend on type of jdbc dialect, we change the index options for test.

Reply:
Can we add a method for this and implement it in each concrete test suite? e.g. `def indexOptions: String = ""`